routers/bearsModel.js

```diff
@@ -0,0 +1,57 @@
+const db = require('../dbConfig')
+
+module.exports = {
+  add,
+  get,
+  getById,
+  remove,
+  update
+}
+
+function get() {
+  return db('bears')
+}
+
+function getById(id) {
+  return db('bears')
+    .where('id', id)
+    .first();
+}
+
+function add(bear) {
+  return db('bears')
+    .insert(bear)
+    .then(ids => {
+      [id] = ids
+      return getById(id)
+    });
+}
+
+function remove(id) {
+  return getById(id)
+    .then(bear => {
+      if (bear) {
+        return db('bears')
```

Comment:
This is all a bit much, I would put the validation on the route, or not at all since .del() will return 0 if the id doesn't exist